backend/routes/amendBooking/changeVideoLinkController.ts

```diff
@@ -30,13 +30,43 @@ export default class ChangeVideoLinkController {
       const errors = req.flash('errors') || []
       const [input] = req.flash('input')
 
+      const update = getUpdate(req)
+
       const [bookingDetails, courts] = await Promise.all([
         this.bookingService.get(res.locals, parseInt(bookingId, 10)),
         this.locationService.getVideoLinkEnabledCourts(res.locals, username),
       ])
 
       const rooms = await this.locationService.getRooms(res.locals, bookingDetails.agencyId)
 
+      const currentBookingDetails = update
+        ? {
+            date: update.date.format(DAY_MONTH_YEAR),
+            courtId: update.courtId,
+            startTimeHours: update.startTime.format(HOURS_TIME),
+            startTimeMinutes: update.startTime.format(MINUTES_TIME),
+            endTimeHours: update.endTime.format(HOURS_TIME),
+            endTimeMinutes: update.endTime.format(MINUTES_TIME),
```

Comment:
guess this is a bit inconsistent - one version using `Hours` and the other `.format(HOURS_TIME)`

Maybe fix in a follow up?

Reply:
would be tempted to extract out into a function in `form.ts` like the `toFormValues` in `backend/routes/createBooking/newBooking/form.ts`.

Reply:
I think I decided to go with this as the update date and times were moments but the bookingDetails date and times were all strings - could convert update values to strings and use Hours/Minutes methods there as well?